src/features/profile/DangerCard.js

```diff
@@ -68,10 +68,10 @@ const DangerCard = () => {
         alert(`Oopsie (${error.message}), please try again.`)
       } else {
         alert(`Success, all your entries were deleted.`)
+        // call trackGoal here, when the customer has succeeded in deleting her/his/their entries
+        trackGoal(FATHOM_DELETE_CONFIRMED);
```

Comment:
**suggestion:** Move above the alert, in case user does not click inside alert and just closes the window.

Reply:
if customer: not click alert & closes window #303
bf3c926